apps/openmw/mwworld/cellstore.hpp

```diff
@@ -184,6 +184,11 @@ namespace MWWorld
                 throw std::runtime_error ("Storage for this type not exist in cells");
             }
 
+            template <class T>
+            CellRefList<T>& getReadOnly() {
+                throw std::runtime_error ("Read Only access not available for this type");
```

Comment:
Please, instead `this type` in message use `typeid(T).name()` -- this is better for debugging.